sdk/rust/oak_tests/src/lib.rs

```diff
@@ -413,16 +410,27 @@ impl OakRuntime {
 impl OakNode {
     fn new() -> OakNode {
         OakNode {
-            next_handle: 1 as oak::Handle,
             contents_name: "<default>".to_string(),
             halves: HashMap::new(),
             ports: HashMap::new(),
             thread_handle: None,
         }
     }
+    fn next_handle(&self) -> oak::Handle {
```

Comment:
This does not seem thread safe (assuming we wanted this to at some point be promoted to be an "actual" runtime): two concurrent calls may (unlikely) result in the same handle being returned. We should probably add a mutex / RWlock on the handles table?

Reply:
I don't think there can be concurrent calls to this &ndash; each `OakNode` object is owned by an `OakRuntime` object, and access to that is already under a `RWLock`.